setup.py

```diff
@@ -16,7 +16,7 @@ def main():
         all_deps += extras[group_name]
     extras['all'] = all_deps
     extras['dev'] = [
-        'pylint==2.4.2',
+        'pylint==2.9.6',
         'pylintfileheader==0.0.2',
```

Comment:
There is `0.3.2` version available. Maybe we should also bump here?

Reply:
good point -- in neptune-client we just have unpinned version, I can do the same here